lib/nodes/webrtc/signaling_message.cpp

```diff
@@ -0,0 +1,211 @@
+/** WebRTC signaling messages.
+ *
+ * @author Steffen Vogel <svogel2@eonerc.rwth-aachen.de>
+ * @copyright 2014-2022, Institute for Automation of Complex Power Systems, EONERC
```

Comment:
Please add yourself as @author as well as another @copyright entry to all WebRTC related files:

```
@author Philipp Jungkamp <philipp.jungkamp@opal-rt.com>
@copyright 2023, OPAL-RT Germany GmbH
```